docs/scalardb-supported-databases.md

```diff
@@ -1,31 +1,141 @@
+{% endcapture %}
+
+<div class="notice--info">{{ notice--info | markdownify }}</div>
+
+| Version | Apache Cassandra 4.1 | Apache Cassandra 4.0 | Apache Cassandra 3.11 | Apache Cassandra 3.0 | Apache Cassandra 2.2 |
+|:------------------|:---------------|:-----------------|:----------------|:-----------------|:----------------|
+| **ScalarDB 3.10** | ❌ | ❌ | ✅ | ✅ | ✅ |
+| **ScalarDB 3.9** | ❌ | ❌ | ✅ | ✅ | ✅ |
+| **ScalarDB 3.8** | ❌ | ❌ | ✅ | ✅ | ✅ |
+| **ScalarDB 3.7** | ❌ | ❌ | ✅ | ✅ | ✅ |
+| **ScalarDB 3.6** | ❌ | ❌ | ✅ | ✅ | ✅ |
+| **ScalarDB 3.5** | ❌ | ❌ | ✅ | ✅ | ✅ |
+| **ScalarDB 3.4** | ❌ | ❌ | ✅ | ✅ | ✅ |
+
+## Azure Cosmos DB for NoSQL
+
+| Version | Azure Cosmos DB for NoSQL |
+|:------------------|:---------------|
+| **ScalarDB 3.10** | ✅ |
+| **ScalarDB 3.9** | ✅ |
+| **ScalarDB 3.8** | ✅ |
+| **ScalarDB 3.7** | ✅ |
+| **ScalarDB 3.6** | ✅ |
+| **ScalarDB 3.5** | ✅ |
+| **ScalarDB 3.4** | ✅ |
+
+## JDBC databases
+
+{% capture notice--info %}
+**Note**
+
+For recommendations when using JDBC databases, see [JDBC database recommendations](requirements/#jdbc-database-requirements).
+{% endcapture %}
+
+<div class="notice--info">{{ notice--info | markdownify }}</div>
+
+### Amazon Aurora MySQL
+
+| Version | Amazon Aurora MySQL 3 | Amazon Aurora MySQL 2 |
+:---------------|:---------------|:---------------|
+| **ScalarDB 3.10** | ✅ | ✅ |
+| **ScalarDB 3.9** | ✅ | ✅ |
+| **ScalarDB 3.8** | ✅ | ✅ |
+| **ScalarDB 3.7** | ✅ | ✅ |
+| **ScalarDB 3.6** | ✅ | ✅ |
+| **ScalarDB 3.5** | ✅ | ✅ |
+| **ScalarDB 3.4** | ✅ | ✅ |
+
+### Amazon Aurora PostgreSQL
+
+| Version | Amazon Aurora PostgreSQL 14 | Amazon Aurora PostgreSQL 13 | Amazon Aurora PostgreSQL 12 |
+:---------------|:---------------|:---------------|:---------------|
+| **ScalarDB 3.10** | ✅ | ✅ | ✅ |
+| **ScalarDB 3.9** | ✅ | ✅ | ✅ |
+| **ScalarDB 3.8** | ✅ | ✅ | ✅ |
+| **ScalarDB 3.7** | ✅ | ✅ | ✅ |
+| **ScalarDB 3.6** | ✅ | ✅ | ✅ |
+| **ScalarDB 3.5** | ✅ | ✅ | ✅ |
+| **ScalarDB 3.4** | ✅ | ✅ | ✅ |
+
+### Microsoft SQL Server
+
+| Version | Microsoft SQL Server 2022 | Microsoft SQL Server 2019 | Microsoft SQL Server 2017 |
+:---------------|:---------------|:---------------|:---------------|
+| **ScalarDB 3.10** | ✅ | ✅ | ✅ |
+| **ScalarDB 3.9** | ✅ | ✅ | ✅ |
+| **ScalarDB 3.8** | ✅ | ✅ | ✅ |
+| **ScalarDB 3.7** | ✅ | ✅ | ✅ |
+| **ScalarDB 3.6** | ✅ | ✅ | ✅ |
+| **ScalarDB 3.5** | ✅ | ✅ | ✅ |
+| **ScalarDB 3.4** | ✅ | ✅ | ✅ |
+
+### MySQL
+
+| Version | MySQL 8.0 | MySQL 5.7 |
+|:---------------|:---------------|:---------------|
+| **ScalarDB 3.10** | ✅ | ✅ |
+| **ScalarDB 3.9** | ✅ | ✅ |
+| **ScalarDB 3.8** | ✅ | ✅ |
+| **ScalarDB 3.7** | ✅ | ✅ |
+| **ScalarDB 3.6** | ✅ | ✅ |
+| **ScalarDB 3.5** | ✅ | ✅ |
+| **ScalarDB 3.4** | ✅ | ✅ |
+
+### Oracle
+
+| Version | Oracle 21.3.0-xe | Oracle 18.4.0-xe |
```

Comment:
@brfrn169 Based on the comment in https://github.com/scalar-labs/scalardb/pull/1012#discussion_r1302374704, I think we should add Oracle 23c to this table. 

Do ScalarDB versions 3.4 to 3.10 all support Oracle 23c?

Reply:
**Note:** See https://github.com/scalar-labs/scalardb/pull/1020#discussion_r1303972487